net2/FlowAggrTool.js

```diff
@@ -231,7 +231,7 @@ class FlowAggrTool {
     let end = options.end;
 
     // if working properly, sumflow should be refreshed in every 10 minutes
-    let expire = options.expireTime || 24 * 60; // by default expire in 24 minutes
+    let expire = options.expireTime || 24 * 60 * 60; // by default expire in 24 hours
```

Comment:
no need to keep sumflow that long, 24 mins should be enough, or the memory usage will goes high.